web/libs/editor/tests/integration/e2e/drafts/submit.cy.ts

```diff
@@ -0,0 +1,66 @@
+import { Choices, LabelStudio, ToolBar } from "@humansignal/frontend-test/helpers/LSF";
+import { FF_DEV_3873, FF_REVIEWER_FLOW } from "../../../../src/utils/feature-flags";
+
+enum RESPONSE_TYPE {
+  DRAFT = "draft",
+  SUBMIT = "submit",
+}
+const RESPONSE_DELAY = 100;
+const WAITING_FOR_EVENTS_DELAY = 200;
+
+describe("Annotation submitting process", () => {
+  it("should not create a duplicate annotation if the annotation and draft are saved concurrently", () => {
+    LabelStudio.addFeatureFlagsOnPageLoad({
+      [FF_DEV_3873]: true,
+      [FF_REVIEWER_FLOW]: true,
+    });
+    // here will be stored an order of completed saving action
+    // it consists of strings: "draft" and "submit"
+    const callOrder: RESPONSE_TYPE[] = [];
+
+    LabelStudio.params()
+      .data({
+        text: "Some words",
+      })
+      .config(`<View>
+  <Text name="text" value="$text" />
+  <Choices toName="text" name="choice">
+    <Choice value="ClickMe"/>
+  </Choices>
+</View>`)
+      // To have "new" annotation we need to use userGenerate
+      .withAnnotation({ userGenerate: true, result: [] })
+      .withEventListener("submitDraft", () => {
+        return new Promise((resolve) => {
+          // initialize saving annotation exactly when request of saving draft should be sent to the server
+          ToolBar.submitBtn.click();
+
+          // this emulates server response delay
+          setTimeout(() => {
+            callOrder.push(RESPONSE_TYPE.DRAFT);
+            resolve(void 0);
```

Comment:
`void 0`? why?
can we just skip this param?

Reply:
Idk. My IDE shows warning if there is nothing. 😅